config/develop/config.yaml

```diff
@@ -4,3 +4,4 @@ raw_bucket_name: recover-dev-raw-data
 template_bucket_name: recover-dev-cloudformation
 intermediate_bucket_name: recover-dev-intermediate-data
 processed_data_bucket_name: recover-dev-processed-data
+gx_reports_bucket_name: recover-dev-gx-reports
```

Comment:
Any preference here @philerooski ?

The URLs would be:
```
https://s3.amazonaws.com/recover-dev-great-expectation-reports/index.html
https://s3.amazonaws.com/recover-dev-great-expectation-reports/sub-folder/index.html
```
vs:
```
https://s3.amazonaws.com/recover-dev-gx-reports/index.html
https://s3.amazonaws.com/recover-dev-gx-reports/sub-folder/index.html
```

Reply:
+1 I like the more explicit name. Maybe we could name it in a more abstracted way, even? We don't necessarily want to expose only GX reports, particularly if this is our "share things to non-devs over the internet" bucket.

Reply:
`recover-dev-data-quality-reports` ?

Reply:
Maybe even something more boring: `recover-dev-vpn-external`.

Reply:
Thoughts on:
`recover-dev-shareable-artifacts-vpn`?

Then if we were to ever host something anonymously accessible we could then name the bucket:
``recover-dev-shareable-artifacts`

Reply:
Both of those work for me. I think the defining trait of the bucket is that it is restricted to VPN access. Since this is all FISMA data, if we were (and I'm not sure we'd be allowed) to make something anonymously accessible, it would be a separate bucket.

Reply:
>  I think the defining trait of the bucket is that it is restricted to VPN access. Since this is all FISMA data, if we were (and I'm not sure we'd be allowed) to make something anonymously accessible, it would be a separate bucket.

Agreed. The template I set up here only give you the option of sharing this with the VPN or not sharing it with the VPN. We'll need to make a new template or modify this later on if we want to share this to a wider audience.

> Both of those work for me.

I'll move forward with `recover-dev-shareable-artifacts-vpn` if that looks good to you @rxu17 ?

Reply:
Yup!